disnake/invite.py

```diff
@@ -147,12 +147,22 @@ class PartialInviteGuild:
         The partial guild's name.
     id: :class:`int`
         The partial guild's ID.
-    verification_level: :class:`VerificationLevel`
-        The partial guild's verification level.
-    features: List[:class:`str`]
-        A list of features the guild has. See :attr:`Guild.features` for more information.
     description: Optional[:class:`str`]
         The partial guild's description.
+    features: List[:class:`str`]
+        A list of features the guild has. See :attr:`Guild.features` for more information.
```

Comment:
```suggestion
        A list of features the partial guild has. See :attr:`Guild.features` for more information.
```
(I know this is unrelated to the PR but since we're here c:)